packages/registry/registry/polkadot-ui/blocks/address-input/address-input.papi.tsx

```
@@ -32,18 +33,15 @@ export function AddressInput(props: AddressInputProps) {
   // Simple services object with type-compatible wrappers
   const services = useMemo<AddressInputServices<ChainIdWithIdentity>>(
     () => ({
-      useIdentity: (address: string, identityChain?: ChainIdWithIdentity) => {
+      useIdentityOf: (address: string, identityChain?: ChainIdWithIdentity) => {
         const chain = identityChain ?? "paseoPeople";
-        return papiUseIdentity(address, chain);
+        return useIdentityOf({ address, chainId: chain });
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)